package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -422,9 +443,10 @@ def guess_donors(self, select='all', max_charge=-0.5):
         Parameters
         ----------
         select: str (optional)
-            Selection string for atom group from which donors will be identified.
+            :doc:`Selection string </documentation_pages/selections>` for atom group from which donors 
```

Comment:
see above, use ref please

Reply:
please also keep within 79 char linewidth (unfortunately, PEP8 checks are currently not running #3854 )

Reply:
I let that slide since the rest of the doc strings didn't adhere.